addons/website/static/tests/tours/conditional_visibility.js

```diff
@@ -268,3 +268,46 @@ registerWebsitePreviewTour("conditional_visibility_5", {
         trigger: ".o_we_invisible_el_panel .o_we_invisible_root_parent.o_we_invisible_entry:contains('Text - Image') + ul .o_we_invisible_entry.o_we_sublevel_1:contains('Column')",
     },
 ]);
+
+wTourUtils.registerWebsitePreviewTour("conditional_visiblity_user_groups", {
+    edition: true,
+    url: "/?debug=1",
+    test: true,
+}, () => [
+    wTourUtils.goBackToBlocks(),
+    {
+        content: "Drag the Title building block and drop it at the bottom of the page.",
+        trigger: "#oe_snippets .oe_snippet[name='Title'].o_we_draggable .oe_snippet_thumbnail:not(.o_we_already_dragging)",
+        extra_trigger: ".o_website_preview.editor_enable.editor_has_snippets",
```

Comment:
```extra_trigger``` is not available now, please adapt tour.

Reply:
same goes here as well.